read-fonts/src/tables/value_record.rs

```diff
@@ -38,16 +38,31 @@ pub struct ValueRecord {
     pub y_placement_device: BigEndian<Nullable<Offset16>>,
     pub x_advance_device: BigEndian<Nullable<Offset16>>,
     pub y_advance_device: BigEndian<Nullable<Offset16>>,
+    #[doc(hidden)]
+    // exposed so that we can preserve format when we round-trip a value record
+    pub format: ValueFormat,
 }
 
-impl ValueRecord {
-    pub fn read_old(data: &[u8], format: ValueFormat) -> Result<Self, ReadError> {
-        let data = FontData::new(data);
-        Self::read(data, format)
+// we ignore the format for the purpose of equality testing, it's redundant
```

Comment:
if it's redundant why do we need to store it? - seems slightly contradictory. Perhaps further explanation of how this makes sense?

Reply:
sorry, it is redundant for the purposes of comparing two value records. In that case (testing, e.g.) I don't think we care.